CONTRIBUTING.md

```diff
@@ -1,46 +1,70 @@
-# How to contribute
+# Contributing to Devopness
 
-Support and contributions from the open source community are essential for keeping
-`@devopness/sdk-js` up to date and always improving! There are a few guidelines that we need
-contributors to follow in order to keep the project consistent and allow us to keep
-maintaining `@devopness/sdk-js` in a reasonable amount of time.
+Thank you for considering contributing to Devopness!
 
-Please note that this project is released with a [Contributor Code of Conduct][coc].
-By participating in this project you agree to abide by its terms.
+We appreciate the help!
 
-[coc]: ./CODE_OF_CONDUCT.md
+All community contributions are welcome! Being them:
+- 🐛 Opening [a bug report](https://github.com/devopness/devopness/issues/new/choose)
+- 📚 Improving [our documentation](https://github.com/devopness/devopness/tree/main/docs)
+- 🆕 Submitting [ideas and feature requests](https://github.com/devopness/devopness/discussions/categories/ideas-feature-requests)
+- ❓ Asking questions and helping other Devopness users with open [discussions](https://github.com/devopness/devopness/discussions)
+... every small contribution matters a lot to our team and for the Devopness community!
 
-## Creating an Issue
+## Not ready to start contributing?
+No problem!
+If you like Devopness, but just don't have time to contribute yet, that's fine.
+There are other easy ways to support Devopness and show your appreciation:
+- ⭐ Star this repository
+- 📣 Post about Devopness on social networks, Discord, LinkedIn, Slack, Twitter, ...
 
-Before you create a new Issue:
+## Code of Conduct
 
-1. Please search existing issues to ensure the same issue hasn't been reported before and avoid opening a duplicate.
-2. If it is a bug report, include the steps needed to reproduce the issue and please create a reproducible test case on [codesandbox.io](https://codesandbox.io/), [jsfiddle.net](https://jsfiddle.net/) or [runkit.com](https://runkit.com/). Example: https://runkit.com/gr2m/5aa034f1440b420012a6eebf
-3. If it is a feature request, please share the motivation for the new feature and how you would implement it.
-4. Please include links to the corresponding GitHub documentation.
+Please make sure to read and observe our [Contributor Code of Conduct](./CODE-OF-CONDUCT.md).
```

Comment:
@jfoliveira, could you fix the broken link OR include the file name change in the PR branch?

```suggestion
Please make sure to read and observe our [Contributor Code of Conduct](./CODE_OF_CONDUCT.md).
```

Reply:
Fixed, thanks!